test/mux/video/assets_test.exs

```diff
@@ -70,6 +70,38 @@ defmodule Mux.Video.AssetsTest do
           }
         }
 
+      %{
+        method: :post,
+        url: @base_url <> "/video/v1/assets/00ecNLnqiG8v00TLqqeZ00uCE5wCAaO3kKc/playback-ids"
+      } ->
+        %Tesla.Env{
+          status: 201,
+          body: %{
+            "data" => Mux.Fixtures.playback_id()
+          }
+        }
+
+      %{
+        method: :get,
+        url:
+          @base_url <>
+              "video/v1/assets/00ecNLnqiG8v00TLqqeZ00uCE5wCAaO3kKc/playback-ids/FRDDXsjcNgD013rx1M4CDunZ86xkq8A02hfF3b6XAa7iE"
```

Comment:
You need a `/` in the front of the string literal here to match the mock. `"/video/v1` not `"video/v1`